doc/website/blog/2022-03-14-ent-hooks-examples.md

```diff
@@ -0,0 +1,430 @@
+---
+title: A few examples for Ent hooks  
+author: Yoni Davidson
+authorURL: "https://github.com/yonidavidson"
+authorImageURL: "https://avatars0.githubusercontent.com/u/5472778"
+authorTwitter: yonidavidson
+---
+Hooks are a popular subject in our blog posts
+[Building Observable Ent Applications with Prometheus](/blog/2021/08/12/building-observable-ent-application-with-prometheus)
+[Sync Changes to External Data Systems using Ent Hooks](/blog/2021/11/1/sync-to-external-data-systems-using-hooks)
```

Comment:
this sentence doesn't compile.. hooks are a popular subject in our blog posts [link] [link] but their reactive nature makes them..? 

Maybe something like,

Despite being one of the most powerful features of Ent, Schema Hooks (link) are often overlooked by many users.  We've covered hooks in previous blog posts (such as x and y), but learning from my personal experience building real applications with Ent, I thought it would be beneficial to put them in the spotlight once more.

Reply:
👍